docs/how-to/test-file-properties.rst

```diff
@@ -51,8 +51,7 @@ Example
             @pytest.fixture(scope='session')
             @dt.working_directory(__file__)
             def df():
-                pattern = r'../*.*'
-                paths = (p for p in pathlib.Path('.').glob(pattern) if p.is_file())
+                paths = (p for p in pathlib.Path('..').glob('*.*') if p.is_file())
```

Comment:
These changes make sense but I want to keep pattern on its own line. Though I agree that the two directory references ("." and "..") is a code smell. I'm going to change this to operate in the current directory, change the pattern to "*.csv", add a comment, and remove the test_file_type() function.

Reply:
OK cool. If you are going to keep the `pattern` variable, may I suggest binding the literal `'.'`  to a variable as well, maybe `dir`? Then it would be quite verbose, clear, and easy to read (and `.` vs `..` would be easy to spot in a line like `dir = "."`).